helpers/spark-submit.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import pwd
+import subprocess
+import logging
+import argparse
+
+USER_HOME_DIR_ENT_IDX = 5
+EXIT_CODE_BAD_KUBECONFIG = -100
+EXIT_CODE_BAD_CONF_ARG = -200
+
+def expand_arg(arg: str) -> str:
+    if os.environ.get(arg):
+        return os.environ[arg]
+    else:
+        return arg
```

Comment:
You could also do simply

`os.environ.get(arg, arg)`

Given it is this simple, you probably don't need a function for this